docs/ERdiag.md

```diff
@@ -0,0 +1,31 @@
+```mermaid
+
+erDiagram
+    USER }|--o{ GROUP : composes
+    GROUP ||--o{ EXPENSE : contains
+    USER ||--o{ EXPENSE : makes
+    USER }|--|{ EXPENSE : "is concerned by"
+
+USER {
+    String username
+    String name 
+    String surname
+    String mail 
+    String phoneNumber 
+    List groups
+}
+    
+GROUP {
+    String name
+    List users 
+    List expenses
```

Comment:
Ici la relation est différente, c'est du `one-to-many` (un group peut avoir plusieurs expenses, une expense ne peut avoir qu'un group). Du coup il est préférable (pour les mêmes raisons que les relations `many-to-many`) de venir stocker la clef étrangère du group  (qui sera généralement son id) dans la table de l'expense qui lui est reliée.

Pareil, comme c'est un diagramme ER pas besoin de faire apparaître cette clef étrangère ici, mais dans un MLD/MPD ça serait le cas.